package-lock.json

```diff
@@ -7808,6 +7906,11 @@
       "integrity": "sha512-7p4Kn/gffhQaavNfyDFg7LS5S/UT1JAjyGd4UqR2+jzoYF02eDkj0Ec3+48TsIa4zghjLY87nQHIh/ecK9qLdw==",
       "dev": true
     },
+    "class-transformer": {
```

Comment:
Thanks so much for cleaning up the package, just make sure to run `npm i` and commit the changes before merging the PR, otherwise the package will still be in the package-lock.json